lib/config.js

```diff
@@ -75,6 +75,11 @@ Config.prototype.buildArgs = function () {
     args.is_win_fastlink = true
   }
 
+  if (this.targetArch === 'x86' && process.platform === 'linux') {
+    // Minimal symbols for target Linux x86, because ELF32 cannot be > 4GiB
```

Comment:
why isn't this an issue for Chrome?

Reply:
there appear to be several conditions in compiler.gni and it looks like this would actually set the symbol level from 0 to 1. If this is targeting android, shouldn't it be covered by `is_android && !is_component_build &&
             !(android_64bit_target_cpu && !build_apk_secondary_abi)` ?